include/Contact.php

```diff
@@ -577,11 +577,6 @@ function get_contact($url, $uid = 0, $no_update = false) {
 
 	require_once('include/Probe.php');
 	$data = Probe::uri($url);
-	if (!$data['url']) {
-		return 0;
-	}
-
-	$url = $data["url"];
 
 	// Does this address belongs to a valid network?
 	if (!in_array($data["network"], array(NETWORK_DFRN, NETWORK_OSTATUS, NETWORK_DIASPORA))) {
```

Comment:
I just realized that the probing now returns data for pump.io based networks as well. So I guess we should include this network here as well.